config/locales/it.yml

```diff
@@ -97,9 +97,12 @@ it:
   disclaimer:
     title: "Organizzazione"
 
-  misc:
+  genders:
     male: "Maschio"
     female: "Femmina"
+    other: 'altro'
```

Comment:
I don't remember who I added the Italian translations for. Anyone speak Italian? @SAuroux, do you know anyone at the WCA that can confirm that this is the correct translation?